Makefile

```diff
@@ -96,12 +91,12 @@ list:
 		docker rm -v $$CID
 
 %.gtfs.tar:
-	set -e ;\
-		ITAG=headway_build_gtfs_$$(echo $(notdir $*) | tr '[:upper:]' '[:lower:]') ;\
-		docker build ./gtfs --build-arg HEADWAY_AREA=$(notdir $*) --tag $${ITAG} ;\
-		CID=$$(docker create $${ITAG}) ;\
-		docker cp $$CID:/gtfs_feeds/gtfs.tar $@ ;\
-		docker rm -v $$CID
+	ITAG=headway_build_gtfs_$$(echo $(notdir $*) | tr '[:upper:]' '[:lower:]')
+	HEADWAY_BBOX=$$(grep "$(notdir $*):" web/bboxes.csv | cut -d':' -f2)
+	docker build ./gtfs_build --build-arg HEADWAY_BBOX="$${HEADWAY_BBOX}" --tag $${ITAG}
+	CID=$$(docker create $${ITAG})
+	docker cp $$CID:/gtfs_feeds/gtfs.tar $@
+	docker rm -v $$CID
```

Comment:
I did the `set -e` removal in my #38 since you mentioned it, FYI. I can take it out since you are making this change, too, if you'd prefer.